### PR TITLE
Update installing-ams-on-linux.md

### PR DESCRIPTION
**Pull request by muratugureminoglu**
### What does it do?

Describe your PR here.

### Why is it needed?

Describe the issue you are solving. It would be great if you reference issue here with brief description.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. If you don't have an issue, please create one.
- [ ] Prefix your PR title with `feat: `, `fix: `, `chore: `, `docs:`, or `refactor:`.
- [ ] The description should clearly illustrate what problems it solves.
- [ ] Ensure that the commit messages follow our guidelines.
- [ ] Resolve merge conflicts (if any).
- [ ] Make sure that the current branch is upto date with the `main` branch.

### CODE REVIEW

File: docs/guides/installing-on-linux/installing-ams-on-linux.md [hunk 1]
```diff
@@ -15,7 +15,38 @@ This document describes how to install both the Community Edition and the Enterp
...
+#### Download the installation script
+```
+wget -O install_ant-media-server.sh https://raw.githubusercontent.com/ant-media/Scripts/master/install_ant-media-server.sh && sudo chmod 755 install_ant-media-server.sh
+```
+#### Install the Enterprise Edition
+
+Add your license key as a parameter, which you received via email, to the installation file using the -l parameter, and then execute it. If you have a valid license key, the installation process will begin.
+
+```
+./install_ant-media-server.sh -l 'your-license-key'
```
Comment (mekya) [inline docs/guides/installing-on-linux/installing-ams-on-linux.md:38]: `sudo` is missing.

File: docs/guides/installing-on-linux/installing-ams-on-linux.md [hunk 2]
```diff
@@ -43,19 +74,19 @@ After you've downloaded or copied the file to the AMS instance, open a terminal
 cd path/to/where/ant-media-server....zip
 ```
 
-## Download the installation script
+#### Download the installation script
 
 Download the ```install_ant-media-server.sh``` shell script.
 
     sudo wget https://raw.githubusercontent.com/ant-media/Scripts/master/install_ant-media-server.sh && sudo chmod 755 install_ant-media-server.sh
```
Comment (mekya) [inline docs/guides/installing-on-linux/installing-ams-on-linux.md:81]: Please put them as code by putting in ``` block

File: docs/guides/installing-on-linux/installing-ams-on-linux.md [hunk 3]
```diff
@@ -15,7 +15,38 @@ This document describes how to install both the Community Edition and the Enterp
...
+```
+./install_ant-media-server.sh -l 'your-license-key'
+```
+
+#### Install the Community Editon
+
+You can simply run the script without any parameters to automatically download and install the latest version of the Community Edition.
+
+```
+./install_ant-media-server.sh
```
Comment (mekya) [inline docs/guides/installing-on-linux/installing-ams-on-linux.md:46]: Please add sudo command

File: docs/guides/installing-on-linux/installing-ams-on-linux.md [hunk 4]
```diff
@@ -43,19 +74,19 @@ After you've downloaded or copied the file to the AMS instance, open a terminal
...
+#### Download the installation script
 
 Download the ```install_ant-media-server.sh``` shell script.
 
     sudo wget https://raw.githubusercontent.com/ant-media/Scripts/master/install_ant-media-server.sh && sudo chmod 755 install_ant-media-server.sh
 
-## Run the installation script
+#### Run the installation script
 
     sudo ./install_ant-media-server.sh -i <ANT_MEDIA_SERVER_ZIP_FILE>
```
Comment (mekya) [inline docs/guides/installing-on-linux/installing-ams-on-linux.md:85]: Please put them as code by putting in ``` block

**Review by mekya — CHANGES_REQUESTED** [review]
I've minor requests. Please check the comments

File: docs/guides/installing-on-linux/installing-ams-on-linux.md (hunk 2 above)
Comment (asharamseervi) [reply docs/guides/installing-on-linux/installing-ams-on-linux.md]: I reviewed about suggested changes. I fixed a few more similar blocks and minor formatting.
Thanks, I'm merging this PR for now.

**Review by asharamseervi — APPROVED** [review]

**Pull request merged**